### PR TITLE
LPS-157563 Adapt Edit ObjectField to show already created filters

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-157563

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/EditObjectField.tsx [hunk 1]
```diff
@@ -365,6 +400,134 @@ export default function EditObjectField({
...
+									(picklistFilterValue) => {
+										return {
+											checked: true,
+											label: picklistFilterValue,
+											value: picklistFilterValue,
+										};
+									}
+								);
+
+								const picklistAggregationfilter: AggregationFilters = {
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/EditObjectField.tsx:510]: This variable should be reamed:
```suggestion
								const picklistAggregationFilter: AggregationFilters = {
```

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**